docs/guides/result_storage.mdx

```diff
@@ -5,106 +5,96 @@ title: Result Storage
 
-Each actor run is associated with a **default key-value store**, which is created exclusively for the actor run. By convention, the actor run input
-and output is stored in the default key-value store under the `INPUT` and `OUTPUT` key, respectively. Typically the input and output is a JSON file,
-although it can be any other format.
+Each Crawlee project is associated with a **default key-value store**. By convention, the project input and output are stored in the default key-value store under the `INPUT` and `OUTPUT` keys respectively. Typically, both input and output are JSON files, although they could be any other format.
 
-In Crawlee, the key-value store is represented by the <ApiLink to="core/class/KeyValueStore">`KeyValueStore`</ApiLink> class. In order to simplify access to the default
-key-value store, Crawlee also provides <ApiLink to="apify/class/Actor#getValue">`Actor.getValue()`</ApiLink> and <ApiLink to="apify/class/Actor#setValue">`Actor.setValue()`</ApiLink> functions.
+In Crawlee, the key-value store is represented by the <ApiLink to="core/class/KeyValueStore">`KeyValueStore`</ApiLink> class. In order to simplify access to the default key-value store, Crawlee also provides <ApiLink to="core/class/KeyValueStore#getValue">`KeyValueStore.getValue()`</ApiLink> and <ApiLink to="core/class/KeyValueStore#setValue">`KeyValueStore.setValue()`</ApiLink> functions.
 
-In local configuration, the data is stored in the directory specified by the `APIFY_LOCAL_STORAGE_DIR` environment variable as follows:
+The data is stored in the directory specified by the `CRAWLEE_STORAGE_DIR` environment variable as follows:
 
 ```
-{APIFY_LOCAL_STORAGE_DIR}/key_value_stores/{STORE_ID}/{KEY}.{EXT}
+{CRAWLEE_STORAGE_DIR}/key_value_stores/{STORE_ID}/{KEY}.{EXT}
 ```
 
-Note that `{STORE_ID}` is the name or ID of the key-value store. The default key value store has ID `default`, unless you override it by setting the
-`APIFY_DEFAULT_KEY_VALUE_STORE_ID` environment variable. The `{KEY}` is the key of the record and `{EXT}` corresponds to the MIME content type of the
-data value.
+:::note
+
+`{STORE_ID}` is the name or the ID of the key-value store. The default key-value store has ID `default`, unless you override it by setting the `APIFY_DEFAULT_KEY_VALUE_STORE_ID` environment variable. The `{KEY}` is the key of the record and `{EXT}` corresponds to the MIME content type of the data value.
+
+:::
 
 The following code demonstrates basic operations of key-value stores:
 
 ```javascript
-// Get actor input from the default key-value store
-const input = await Actor.getInput();
+import { KeyValueStore } from '@crawlee/core';
+//                   ... from '@crawlee/cheerio';
+//                   ... from '@crawlee/playwright';
+//                   ... from '@crawlee/puppeteer';
```

Comment:
just FYI we will be using the `crawlee` metapackage in docs to simplify things, but you can keep it here, we can do a quick find/replace and fix it in the whole docs in another PR